packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md

```diff
@@ -0,0 +1,50 @@
+---
+id: Timestamp
+section: components
+---
+
+## Elements
+
+<img src="./img/timestamp-with-tooltip-underline.png" alt="Timestamp with tooltip underline"/>
```

Comment:
Need to add width to these images bc they're super blown up right now... should be the width of the artboard they're on in sketch!

Reply:
no problem. We should update the size in the design documentation bc it only states a minimum size. 
![image](https://user-images.githubusercontent.com/91554933/197587562-f2419658-8d09-40de-a815-dea14c9857c2.png)